SPECS/qemu.spec

```
@@ -198,6 +201,10 @@ cp -r scripts/qmp %{buildroot}%{_datarootdir}/qemu
 %{?_cov_results_package}
 
 %changelog
+* Tue Feb 04 2025 Lucas Ravagnier <lucas.ravagnier@vates.tech> - 4.2.1-4.6.4.2
```

Please update the date
```suggestion
* Mon Feb 10 2025 Lucas Ravagnier <lucas.ravagnier@vates.tech> - 4.2.1-4.6.4.2
```